dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1718,6 +1727,15 @@ public void delete(List<Contentlet> contentlets, User user, boolean respectFront
             if(!canLock(con, user)){
                 throw new DotContentletStateException("Content Object is locked and cannot be deleted:" + con.getIdentifier());
             }
+
+            if (con.getInode().equals(CAN_T_CHANGE_STATE_OF_CHECKED_OUT_CONTENT)) {
+                throw new DotDataException("");
+            }
+        }
+
+
+          if (true){
+                System.out.println("ddd");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)